libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/DropInNavigationViewModel.kt

```diff
@@ -21,25 +26,45 @@ internal class DropInNavigationViewModel : ViewModel() {
     private val _navigationState = MutableStateFlow<NavigationState>(NavigationState.FreeDrive)
     val navigationState = _navigationState.asStateFlow()
 
+    @Suppress("PropertyName")
+    val _activeNavigationStarted = MutableStateFlow(false)
+    val activeNavigationStarted = _activeNavigationStarted.asStateFlow()
+
+    private val _onBackPressedEvent = MutableSharedFlow<Unit>(
```

Comment:
Makes sense to throw this in a BackPressViewModel

Reply:
I leave BackPress handling to you. This is just a temp solution to get back press working.